src/main/java/org/rundeck/client/tool/commands/Jobs.java

```diff
@@ -300,6 +316,18 @@ public void info(InfoOpts options, CommandOutput output) throws IOException, Inp
         outputJobList(options, output, Collections.singletonList(body));
     }
 
+    @Command(description = "Get Schedule Forecast for a Job by ID (API v31)")
+    public void forecast(ForecastOpts options, CommandOutput output) throws IOException, InputError {
+        ForecastJobItem body = apiCall(api -> api.getJobForecast(options.getId(), options.getDays(), options.getMax()));
```

Comment:
Should require api version 31:

```suggestion
        requireApiVersion("jobs forecast", 31);
        ForecastJobItem body = apiCall(api -> api.getJobForecast(options.getId(), options.getDays(), options.getMax()));
```